cmd/utils/flags.go

```diff
@@ -2148,6 +2163,30 @@ func IsNetworkPreset(ctx *cli.Context) bool {
 	return false
 }
 
+func DialRPCWithHeaders(endpoint string, headers []string) (*rpc.Client, error) {
+	if endpoint == "" {
+		return nil, errors.New("endpoint must be specified")
+	}
+	if strings.HasPrefix(endpoint, "rpc:") || strings.HasPrefix(endpoint, "ipc:") {
```

Comment:
Can we drop this backward compatibility code?

Reply:
Not sure if we can... we never told people _not_ to use that format, afaik